lib/stylesheet/importer.rb

```diff
@@ -64,9 +64,32 @@ def font
 
         CSS
 
+      # conditionally enable Jetbrains Mono font on rich editor
```

Comment:
I think we can drop this comment, it's clear enough without it

Reply:
sure